composite/market-closure/README.md

```diff
@@ -26,8 +25,8 @@ The adapter takes the following environment variables:
 | :-------: | :-------------------------------: | :-------------------------------------------------------------------------------------------: | :-----: | :---------: |
 |    ✅     |  `referenceContract`, `contract`  |                 The Aggregator contract to call for its latest round's price                  |         |             |
 |    ✅     |            `multiply`             | To handle big numbers, the amount to divide the output from reading the reference contract by |         |             |
-|    ✅     | `base`, `asset`, `from`, `symbol` |                                 The target currency to query                                  |         |             |
 |    🟡     |            `schedule`             |                   A schedule of market times to check whether they are open                   |         |             |
+|    ✅     |                                   |                         Any params required by the underlying adapter                         |         |             |
```

Comment:
I think they will also get confused here. Maybe separate it a little bit. [Here's how I structured mine](https://github.com/smartcontractkit/external-adapters-js/blob/473d8e328be93c2f271ffe6fa74608ad748c292d/composite/proof-of-reserves/README.md)